apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/ResourceNames.cs

```diff
@@ -337,7 +337,7 @@ public partial class Session
         /// </summary>
         public SessionName SessionName
         {
-            get { return string.IsNullOrEmpty(Name) ? null : Google.Cloud.Spanner.V1.SessionName.Parse(Name); }
+            get { return String.IsNullOrEmpty(Name) ? null : SessionName.Parse(Name); }
```

Comment:
This code is autogenerated, and shouldn't be touched by anything. (We will need to regenerate regularly.) In this case, my guess is that it was R# being over-enthusiastic...